### ISSUE TITLE
Import data from splitwise.

### ISSUE THREAD

**Issue by KMKoushik** [oss-apps/split-pro#29]
Currently export option is only available for splitwise pro users. Need to figure out way to make it wotk for everyone!

**Comment by dfego** [comment]
Hi there! So there is an "Export as spreadsheet" option in Splitwise for each group. It's kind of annoying to find, but it exports as a csv file. Accessible on the web version if you go to a group, then click the gear icon on the right side, then "Export as spreadsheet".

That would seem to be the most straightforward way to let anyone import their data!

**Comment by KMKoushik** [comment]
@dfego Yeah, I saw this. Did not like this way though. I have 3 main issues with this approach. 

- I have to manually import for each group.
- Then I have to manually assign Person 1, Person 2 etc to splitpro account
- For non-group balance there's no export option. 

I have a sample PR here: https://github.com/oss-apps/split-pro/pull/37

This will fetch all the group and non group balances. Basically create 100% import with very less efforts. I found a way to download splitwise data. You can test it here: https://export-splitwise.vercel.app/ 

This PR is almost done! With some UI changes and testing, should be available soon!

**Comment by dfego** [comment]
Interesting! I just downloaded mine. They certainly make you jump through a lot of hoops to get it.

Can't wait to see this get in, then I can try this out!

**Comment by josefhelie** [comment]
Hi and thanks for your work! I’m in the process of onboarding to SplitPro and just exported a dataset from Splitwise to convert my tricount data to that format. However, I just realized that only balances were exported, not the individual transactions.

Is there a way to import the full transaction history? My girlfriend and I would love to keep all the history—and all the little jokes we wrote while spending our money! 😊
